front-end/src/components/UploadForm.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+
+function UploadForm(){
+  const [verificationToken, setVerificationToken] = useState("");
+
+  function validToken() {
+    const validTokenCombination = /^[A-Z]+$/;
+    const validTokenLength = 4;
+    if(verificationToken.match(validTokenCombination) && verificationToken.length === validTokenLength)
+    {
+      return true;
+    }
+    return false;
+  }
```

Comment:
@cskeogh I have addressed the maintainability issue by creating a new function that checks the validity of the token.